packages/components/src/components/modals/Modal/Modal.tsx

```diff
@@ -181,6 +191,7 @@ const Modal = ({
     subheading,
     modalPrompt,
     headerIcon,
+    headingSize,
```

Comment:
So you don't need the `??` later

```suggestion
    headingSize = 'default',
```

Reply:
:heavy_check_mark: